### DISCUSSION TITLE
What famous UK castle was built on an extinct volcano?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#176] [Galaxy Trivia] [answered]
**Hint:** in Scotland

**Comment by thuongtruong1009** [comment, marked as answer]
The famous UK castle that was built on an extinct volcano is Edinburgh Castle. It is located in Edinburgh, Scotland, and sits atop Castle Rock, an extinct volcano that rises over 130 meters above sea level. The castle has a rich and fascinating history, dating back to the 12th century, and it has served as a royal residence, military stronghold, and a national symbol of Scotland. Today, Edinburgh Castle is one of Scotland's top tourist attractions and is open to the public year-round.

**Reply by ParanoidUser** [reply]
Salut! 😋 Congrats on your well deserved [x3 Galaxy Brain](https://github.com/Faiqlatheef?tab=achievements&achievement=galaxy-brain) badge! 16 more answers for the gold [x4]. Keep it up!

**Reply by thuongtruong1009** [reply]
@ParanoidUser ❤ How wonderful. Thanks for your wishes. Thanks for your and everyone's help.

**Comment by nguyenhoanghai1605** [comment]
Hi @ParanoidUser,
The famous British castle built on an extinct volcano is Edinburgh Castle. According to Scotland travel and tourism information sites, the castle is located on top of Castle Rock, an ancient volcanic rock.
